docs/_docs/reference/dropped-features/nonlocal-returns.md

```diff
@@ -5,7 +5,7 @@ title: "Deprecated: Nonlocal Returns"
 nightlyOf: https://docs.scala-lang.org/scala3/reference/dropped-features/nonlocal-returns.html
 ---
 
-Returning from nested anonymous functions has been deprecated, and will produce a warning from version `3.2`.
+Returning from nested anonymous functions is deprecated.
```

Comment:
maybe add a comment that this was since 3.2, in case we want to also back create versions since 3.0.0